docs/settings/canned-message-plugin.md

```diff
@@ -173,28 +178,26 @@ Configuring this setting is not yet available for the selected platform. If this
   </TabItem>
 </Tabs>
 
-### canned_message_plugin_send_bell
+### --set-canned-message
```

Comment:
If we go with the above comment about adding the item to the table, I think this should be changed to `messages`